bindings/python/iota_sdk/types/block/block.py

```diff
@@ -2,53 +2,60 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from __future__ import annotations
-from enum import Enum
+from enum import Enum, IntEnum
 from dataclasses import dataclass
-from typing import List, Optional, Union, Dict
-from dacite import from_dict
+from typing import List, Optional
 from iota_sdk.types.common import HexStr, json
-from iota_sdk.types.payload import TaggedDataPayload, TransactionPayload
-from iota_sdk.utils import Utils
+
+
+class BlockType(IntEnum):
+    """Block types.
+
+    Attributes:
+        Basic (1): A Basic Block.
+        Validation (2): A Validation Block.
+    """
+    Basic = 1
```

Comment:
Should this be 0 and 1?

Reply:
apparently I picked the only enum type that started with a 1 as my guide

Reply:
Changed